Source/CsDebugScript.Engine/Engine/Native/DEBUG_LAST_EVENT_INFO.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace CsDebugScript.Engine.Native
+{
+    public enum DEBUG_EVENT
+    {
+        /// <summary>
+        /// A breakpoint exception occurred in the target.
+        /// </summary>
+        DEBUG_EVENT_BREAKPOINT,
```

Comment:
Capital letters are not needed for enums in C#. Also, you don't need to repeat enum name inside the value name.